packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -194,6 +197,132 @@ class WebKitWebViewPlatformController extends WebViewPlatformController {
     };
   }
 
+  @override
+  Future<void> loadHtmlString(String html, {String? baseUrl}) {
+    return webView.loadHtmlString(html, baseUrl);
+  }
+
+  @override
+  Future<void> loadFile(String absoluteFilePath) async {
+    await webView.loadFileUrl(
+      absoluteFilePath,
+      path.dirname(absoluteFilePath),
+    );
+  }
+
+  @override
+  Future<void> loadFlutterAsset(String key) async {
+    assert(key.isNotEmpty);
+    return webView.loadFlutterAsset(key);
+  }
+
+  @override
+  Future<void> loadUrl(String url, Map<String, String>? headers) async {
+    final NSUrlRequest request = NSUrlRequest(
+      url: url,
+      allHttpHeaderFields: headers ?? <String, String>{},
+    );
+    return webView.loadRequest(request);
+  }
+
+  @override
+  Future<void> loadRequest(WebViewRequest request) async {
+    if (!request.uri.hasScheme) {
+      throw ArgumentError('WebViewRequest#uri is required to have a scheme.');
+    }
+
+    final NSUrlRequest urlRequest = NSUrlRequest(
+      url: request.uri.toString(),
+      allHttpHeaderFields: request.headers,
+      httpMethod: describeEnum(request.method),
+      httpBody: request.body,
+    );
+
+    return webView.loadRequest(urlRequest);
+  }
+
+  @override
+  Future<bool> canGoBack() => webView.canGoBack;
+
+  @override
+  Future<bool> canGoForward() => webView.canGoForward;
+
+  @override
+  Future<void> goBack() => webView.goBack();
+
+  @override
+  Future<void> goForward() => webView.goForward();
+
+  @override
+  Future<void> reload() => webView.reload();
+
+  @override
+  Future<String> evaluateJavascript(String javascript) async {
+    return runJavascriptReturningResult(javascript);
```

Comment:
This passthrough doesn't look like it will have the right behavior on older versions of iOS. Prior to 14, when we had to split out runJavascript/runJavascriptReturningResult, evaluateJavascript was the method to call regardless of whether there was a result. You're calling a method that throws on null return.